chalice/constants.py

```diff
@@ -70,3 +70,142 @@ def index():
     ],
     "Resource": "arn:aws:logs:*:*:*"
 }
+
+
+CODEBUILD_POLICY = {
+    "Version": "2012-10-17",
+    # This is the policy straight from the console.
+    "Statement": [
+        {
+            "Action": [
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream",
+                "logs:PutLogEvents"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "s3:GetObject",
+                "s3:GetObjectVersion",
+                "s3:PutObject"
+            ],
+            "Resource": "arn:aws:s3:::*",
+            "Effect": "Allow"
+        }
+    ]
+}
+
+CODEPIPELINE_POLICY = {
```

Comment:
So is this the minimal amount of permissions needed to get codepipeline working? Seems a little weird that there is a good amount of services included in it that don't really need to be like beanstalk and opsworks